### PR TITLE
feat

### PR DESCRIPTION
**Pull request by TKRchamak**
folder structured for backend and frontend

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,23 +1,248 @@
...
+# Edit at https://www.toptal.com/developers/gitignore?templates=django,react,macos,windows
 
-# dependencies
-/node_modules
-/.pnp
-.pnp.js
+### Django ###
+*.log
+*.pot
+*.pyc
```
Comment (Biplob14) [inline .gitignore:7]: why this changed?

**Review by Biplob14 — CHANGES_REQUESTED** [review]
see the comment

File: .gitignore (hunk 1 above)
Comment (TKRchamak) [reply .gitignore:7]: it was necessary

**Review by TKRchamak — COMMENTED** [review]
feadback

**Review by Biplob14 — APPROVED** [review]

**Pull request merged**